docs/modules/ROOT/pages/configuration-store.adoc

```diff
@@ -0,0 +1,172 @@
+<<Usage>> section below.
+
+Here is an example of a base properties file named `messaging.properties`:
+[source,properties]
+----
+connector=smallrye-kafka
+topic=baseTopic
+----
+This file defines two properties, `connector` and `topic`, each with the group name `messaging`.
+
+===== Environment Specific Overrides
+In addition to the base properties file introduced above, one can optionally specify a secondary properties file that houses environment
+specific configurations. These configurations will then used to override their respective base configurations.
+
+Continuing with the example above, suppose one defines an additional properties file named `messaging.properties` like the following:
+[source,properties]
+----
+topic=ciTopic
+auth-token=auth-secret
+----
+
+Then when these two properties files are reconciled within the store, it results in a configuration equivalent to the following:
+[source,properties]
+----
+connector=smallrye-kafka
+topic=ciTopic
+auth-token=auth-secret
+----
+
+NOTE: The `topic` property is overwritten when the environment specific value `ciTopic`.
+
+==== Mounting Properties
+In order for the store to read the configurations, the properties files will need to be placed into a https://kubernetes.io/docs/concepts/storage/persistent-volumes/#lifecycle-of-a-volume-and-claimpersistent[Persistent Volume,role=external,window=_blank] (PV)
+mounted to the configuration store kubernetes resource. It is assumed a PV will meet most projects needs and is the most widely accepted 
+structured solution. The configuration service will then consume these properties files and place them into the configuration store. Please 
+see the configuration-store https://github.com/boozallen/aissemble/blob/{git-tree}/extensions/extensions-helm/aissemble-configuration-store-chart/README.md#persistent-volume-properties[helm chart documentation,role=external,window=_blank] for more information on mounting the PV to the service.
+
+===== Properties File Locations
+Within the configuration store's helm chart, one can specify environment variables with the URIs that specify where the project's various 
+properties files are located within the PV. It is required to specify a base property URI that houses the base/default property 
+configurations. In addition to the base URI, one can optionally specify a secondary URI for properties files that house the environment
+specific configurations. 
+
+NOTE: Multiple configuration properties files can be stored at either URI.
+
+Since it is common practice to define separate Helm chart values for each environment of a project's development lifecycle, it is 
+encouraged to define one shared base URI and respective environment specific URIs, each housing the relevant overrides.
+
+The following Helm chart configuration demonstrates the URI specifications for an example CI deployment:
+[source,yaml]
+----
+  env:
+    BASE_PROPERTY: <URI housing base property configurations>
```

Comment:
S: This is an example of the type of detail that I think is maybe too much for these docs.  There are two risks with this example: 1) it muddies the waters of the overall explanation of the feature and generally how it works, and 2) it's very likely to fall out of date as the chart changes.

Reply:
Moved technical portion to helm chart README